src/plugins/intel_gna/src/common/graph_utils.hpp

```diff
@@ -252,6 +267,10 @@ inline bool is_gna_precision_agnostic(std::shared_ptr<ngraph::Node> node) {
             ((std::dynamic_pointer_cast<ngraph::op::CropIE>(node) != nullptr) && !is_crop_affined(node)));
 }
 
+inline bool is_gna_not_quantizable(std::shared_ptr<ngraph::Node> node) {
```

Comment:
is_node_not_quantizable_by_gna()?

Reply:
Function moved to CPP according to Kuba's comment.